src/index.ts

```diff
@@ -1,10 +1,13 @@
-module.exports = {
+import { TranslationMessages } from "ra-core";
+
+const spanishMessages: TranslationMessages = {
   ra: {
     action: {
       add: "Añadir",
       add_filter: "Añadir filtro",
       back: "Ir atrás",
-      bulk_actions: "1 item seleccionado |||| %{smart_count} items selecccionados",
+      bulk_actions:
+        "1 item seleccionado |||| %{smart_count} items selecccionados",
```

Comment:
Can you fix this word `selecccionados` by `seleccionados` please? Was bad written by us.

Reply:
done!